test/t8_schemes/t8_gtest_face_descendant.cxx

```diff
@@ -103,7 +102,13 @@ class class_descendant: public TestDFS {
     scheme->element_destroy (eclass, 1, &scheme_face_desc);
     dfs_test_teardown ();
   }
-  int max_test_lvl;
+#if T8CODE_TEST_LEVEL == 0
+  int additional_test_lvl = 1;
+#elif T8CODE_TEST_LEVEL == 1
+  int additional_test_lvl = 2;
+#elif T8CODE_TEST_LEVEL == 2
+  int additional_test_lvl = 3;
+#endif
```

Comment:
These numbers should be the other way around.

Level 0 is the most exhausting test level and level 2 the least exhausting one.

I was confused by this as well and we should assign proper names to the test levels.